doc/radosgw/frontends.rst

```diff
@@ -135,6 +135,17 @@ Options
 :Default: ``16384``
 :Maximum: ``65536``
 
+``so_reuseport``
+
+:Description: If set allows multiple rgw instances to run on same port.
```

Comment:
Suggest

:Description: If set allows multiple RGW instances on a host to listen on the same TCP port.

Reply:
Below v s/rgw/RGW/

Reply:
Done.